src/api_endpoints.py

```diff
@@ -169,9 +169,11 @@ def get_scan(self, scan_path):
 
     def get_scan_for_scan_version_workflow_id(
             self, scan_version, workflow_id, scan_path):
-        '''Get the scan metadata'''
+
+        '''Get the scan metadata filtered by scan_version and workflow_id'''
         headers = self.prepare_header()
-        # TODO use scan_version and workflow id
+        # use scan_version and workflow id to get filtered scans
+
         mod_scan_endpoint = self.mod_scan_endpoint + '&scan_version=' + scan_version
```

Comment:
can we create request with params
requests.get(url, params={'scan_version': scan_version, 'page': 1, 'limit': 1})
instead of modifying in endpoint and url

Reply:
Fixed